scss/atoms/Meter/_Meter.scss

```diff
@@ -0,0 +1,58 @@
+// ==========================================================================
```

Comment:
Have file name and folder be spelt with a lower case 'm'

Although that being said, maybe meters should be included with typography?  Thoughts?